src/rgw/support/rgw_support_bundle

```diff
@@ -107,10 +106,23 @@ class RGWSupportBundle:
                 outfile = os.path.join(tmp_motr_trace_dir, file)
                 # Convert m0trace file to human readable yaml format
                 SimpleProcess(f'm0trace -i {infile} -Y -o {outfile}.yaml').run()
+                SimpleProcess(f'xz {outfile}.yaml').run() # compress the output file
 
         if stacktrace:
-            # TODO: CORTX-32151 Generate live stack trace for rgw process
-            pass
+            from subprocess import check_output
+            process = '/usr/bin/radosgw'
+            pid = check_output(['pidof', process]).decode('utf-8').split('\n')[0]
+            stacktrace_file = os.path.join(RGWSupportBundle._tmp_src, 'rgw_live_callstack.log')
+            # using gdb tool with "thread apply all bt full" command to get
+            # stack trace for the process and all the treads
```

Comment:
typo treads -> threads

You can modify to

# Collecting running RGW process ($pid) stack trace using GDB